ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/graphSearch/stepChecking/FootstepPoseHeuristicChecker.java

```diff
@@ -146,64 +143,24 @@ else if (Math.abs(stepHeight.getValue()) > maximumStepZ)
          return BipedalFootstepPlannerNodeRejectionReason.STEP_TOO_HIGH_OR_LOW;
       }
 
-      double alphaPitchedBack = Math.max(0.0, - stanceFootPose.getPitch() / parameters.getMinimumSurfaceInclineRadians());
-      minZFromPitchContraint.set(InterpolationTools.linearInterpolate(Math.abs(maximumStepZ), Math.abs(parameters.getMinimumStepZWhenFullyPitched()), alphaPitchedBack));
-      maxXFromPitchContraint.set(InterpolationTools.linearInterpolate(Math.abs(parameters.getMaximumStepReach()), parameters.getMaximumStepXWhenFullyPitched(), alphaPitchedBack));
-      double stepDownFraction = - stepHeight.getValue() / minZFromPitchContraint.getValue();
-      double stepForwardFraction = stepLength.getValue() / maxXFromPitchContraint.getValue();
+      double alphaPitchedBack = Math.max(0.0, - stanceFootPose.getPitch() / parameters.getMinSurfaceIncline());
 
       stepIsPitchedBack.set(alphaPitchedBack > 0.0);
-      stepTooLow.set(stepLength.getValue() > 0.0 && stepDownFraction > 1.0);
-      stepTooForward.set(stepHeight.getValue() < 0.0 && stepForwardFraction > 1.0);
+      stepTooLow.set(stepLength.getValue() > 0.0 && stepHeight.getValue() > 1.0);
+      stepTooForward.set(stepHeight.getValue() < 0.0 && stepLength.getValue() > 1.0);
```

Comment:
do these do anything anymore? As written, "stepTooLow" is now only true if you're stepping forward and up a meter in the air, which wasn't the intent of that. Also stepTooForward is a similar thing. So this should be either fixed, or these fields should be deleted and the `STEP_TOO_LOW_AND_FORWARD_WHEN_PITCHED` check should be removed.